gqueries/modules/etmoses/number_of_residences.gql

```diff
@@ -0,0 +1,2 @@
+- query = AREA(number_of_new_residences)
+- unit = number
```

Comment:
```diff
  | - query = AREA(number_of_new_residences)
- | - unit = number
+ | - unit = #
```

(I do prefer `number` and there are some inputs which use it, but at the moment `#` is the unit for queries which represent a raw numbers).